### ISSUE TITLE
Scrollable element is not keyboard accessible

### ISSUE THREAD

**Issue by christhepianist** [samvera/hyku#3079]
vpat.hykudemo.org
Version 7.1

### Descriptive summary
This page has a scrollable section that can't be operated by keyboard.:
<img width="1130" height="888" alt="Image" src="https://github.com/user-attachments/assets/1312c134-673b-422a-bede-0a4b7afdf9d0" />

### Rationale
Accessibility issue remediation

Keyboard accessibility is an essential component of an accessible website.

Visitors who can't easily use a mouse may instead use a keyboard (or keyboard alternative) for navigation. This includes people who are blind and people with motor disabilities.

### Expected behavior
Keyboard users should be able to navigate to and edit this section.

### Actual behavior
Keyboard users can navigate to and edit this section.

### Steps to reproduce the behavior
- Open the page
- Use the SiteImprove browser extension

**Comment by christhepianist** [comment]
As a group, we discussed this issue. As a user, I can tab into this area, navigate, and edit. We feel that this sufficiently meets the accessibility guidelines unless someone gives us more information. Therefore, I am closing this issue.
